src/python/doc/rips_complex_user.rst

```diff
@@ -347,3 +347,51 @@ until dimension 1 - one skeleton graph in other words), the output is:
     points in the persistence diagram will be under the diagonal, and
     bottleneck distance and persistence graphical tool will not work properly,
     this is a known issue.
+
+Weighted Rips Complex
+---------------------
```

Comment:
I think we want in this section a link to the reference doc (something like `` `WeightedRipsComplex <rips_complex_ref.html#weighted-rips-complex-reference-manual>`_`` or ``:class:`~gudhi.weighted_rips_complex.WeightedRipsComplex` ``, in the middle of an existing sentence or a new one), and it wouldn't hurt to repeat the first sentence of the reference manual here, with the reference to the paper.